public/v4/apps/teslamate.yml

```diff
@@ -0,0 +1,105 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        image: postgres:$$cap_POSTGRES_VERSION
+        restart: always
+        environment:
+            POSTGRES_DB: $$cap_DATABASE_NAME
+            POSTGRES_USER: $$cap_DATABASE_USER
+            POSTGRES_PASSWORD: $$cap_DATABASE_PASS
+        volumes:
+            - $$cap_appname-db:/var/lib/postgresql/data
+    $$cap_appname-grafana:
+        caproverExtra:
+            containerHttpPort: 3000
+            websocketSupport: 'true'
+        image: teslamate/grafana:$$cap_GRAFANA_VERSION
+        restart: always
+        environment:
+            DATABASE_USER: $$cap_DATABASE_USER
+            DATABASE_PASS: $$cap_DATABASE_PASS
+            DATABASE_NAME: $$cap_DATABASE_NAME
+            DATABASE_HOST: srv-captain--$$cap_appname-db
+        volumes:
+            - $$cap_appname-db:/var/lib/postgresql/data
+        depends_on:
+            - $$cap_appname-db
+    $$cap_appname-mqtt:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        image: eclipse-mosquitto:2
+        restart: always
+        command: mosquitto -c /mosquitto-no-auth.conf
```

Comment:
`command` isn't consumed by CapRover. If this is necessary, please look at the `redis` template for a workaround.

Reply:
Thank you, i noticed that indeed having issues with MQTT with my previous configuration.
That has been resolved now.